hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -190,6 +190,9 @@ public Operation convert(String value) throws ParameterException {
     @Parameter(names = {"--table-type"}, description = "Type of table. COPY_ON_WRITE (or) MERGE_ON_READ", required = true)
     public String tableType;
 
+    @Parameter(names = {"--table-file-format"}, description = "BaseFileFormat of table. PARQUET (or) ORC")
```

Comment:
adding something like this without actual ORC support feel bit premature and misleading to me.. for e,g if we release the code in few weeks, deltastreams help will be very misleading.. let's wait till we have datasource or some level of progress with ORC atleast ?

Reply:
Reasonable, done